input/people_test.csv

```diff
@@ -0,0 +1,16 @@
+age,homeId
```

Comment:
Is this an existing file format? If not:

* Is there a reason to use camelcase here, thus forcing it in serde?
* I would normally think homeId would come first.

Obviously, if this is existing then we have to live with it I guess.

Reply:
This is the format of the ASPRs synthetic populations. We are ignoring here in this example the rest of the fields. I think we can change that, but that's the reason for now.

Reply:
Fair enough.